src/logic/router.js

```diff
@@ -0,0 +1,57 @@
+const get = require('lodash.get');
+const { wrap } = require('lambda-async');
+const Router = require('route-recognizer');
+const { wrap: wrapHandler } = require('./handler');
```

Comment:
Because of the naming collision here, what do you think of importing the packages themselves and using the wrap below?
```
const lambdaAsync = require('lambda-async');
/* ... */
const handler = require('./handler');
/* ... */
  const hdl = lambdaAsync.wrap(async (event, context) => {
    /* ... */
    return handler.wrap(() => { /* ... */ })

Reply:
I agree that there is possibility for confusion here. Not sure what the best way to solve that is yet...

Reply:
Separated this out into `api-gateway.js`. Let me know what you think